src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -2075,7 +2086,179 @@ let rectifyTermColumns () =
         }
     )
 
-// Old stuff, mostly deprecated
+/// <summary>
+/// Tries to get the name of the top level excel worksheet of top level metadata
+/// </summary>
+let tryGetTopLevelMetadataSheetName (context:RequestContext) =
+    promise {
+        let worksheets = context.workbook.worksheets
+
+        let _ = worksheets.load(propertyNames = U2.Case2 (ResizeArray[|"values"; "name"|]))
+
+        do! context.sync().``then``(fun _ -> ())
+
+        let worksheetItems = worksheets.items |> Array.ofSeq
+
+        let metadata =
+            worksheetItems
+            |> Array.choose (fun item -> if item.name.Contains("isa_") then Some item else None)
+
+        let potMetadata =
+            if metadata.Length < 1 then None
+            elif metadata.Length > 1 then failwith "More than one metadata sheet has been found! This cannot be! Please report this as a bug to the developers.!"
+            else Some metadata.[0].name
+
+        return potMetadata
+    }
+
+/// <summary>
+/// Creates excel worksheet with name for top level metadata
+/// </summary>
+/// <param name="name"></param>
+let createTopLevelMetadata name =
+    Excel.run(fun context ->
+        promise {
+            let workSheetName = $"{name}"
+            let newWorkSheet = context.workbook.worksheets.add(workSheetName)
+            newWorkSheet.activate()
+            do! context.sync().``then``(fun _ -> ())
+            let! result = tryGetTopLevelMetadataSheetName context
```

Comment:
remove this check and instead wrap the whole code block starting with `newWorkSheet.activate()` in a try ... with

Reply:
fixed